config/locales/en.yml

```diff
@@ -1937,8 +1938,8 @@ en:
     terms_of_service_changed:
       agreement: By continuing to use %{domain}, you are agreeing to these terms. If you disagree with the updated terms, you may terminate your agreement with %{domain} at any time by deleting your account.
       changelog: 'At a glance, here is what this update means for you:'
-      description: 'You are receiving this e-mail because we''re making some changes to our terms of service at %{domain}. We encourage you to review the updated terms in full here:'
-      description_html: You are receiving this e-mail because we're making some changes to our terms of service at %{domain}. We encourage you to review the <a href="%{path}" target="_blank">updated terms in full here</a>.
+      description: 'You are receiving this e-mail because we''re making some changes to our terms of service at %{domain}. These updates will take effect %{date}. We encourage you to review the updated terms in full here:'
+      description_html: You are receiving this e-mail because we're making some changes to our terms of service at %{domain}. These updates will take effect <strong>%{date}</strong>. We encourage you to review the <a href="%{path}" target="_blank">updated terms in full here</a>.
```

Comment:
This changes the required interpolations for the strings and could cause issues with other locales.